.gitmodules

```diff
@@ -17,3 +17,6 @@
 [submodule "third_party/nanopb"]
 	path = third_party/nanopb
 	url = https://github.com/nanopb/nanopb.git
+[submodule "tools/thrift"]
+	path = tools/thrift
+	url = https://github.com/chedeti/thrift.git
```

Comment:
Why is the module specified like this? If it's your thrift code generation system, then that belongs as actual contents of the repository in tools/thrift , not as a submodule.